cypress/support/commands.js

```diff
@@ -140,3 +140,63 @@ Cypress.Commands.add(
     );
   },
 );
+
+/**
+ * Creates an article.
+ *
+ * @param {string} title The title of an article.
+ * @param {Array<string>} [tags=[]] The tags of an article.
+ * @param {string} [content=''] The content of the article.
+ * @param {boolean} [published=true] Whether or not an article should be published.
+ * @param {string} [description=''] The description of the article.
+ * @param {string} [canonicalUrl=''] The canonical URL for the article.
+ * @param {string} [series=''] The series the article is associated with.
+ * @param {string} [allSeries=[]] The list of available series the article can be a part of.
+ * @param {string} [organizations=[]] The list of organizations the author of the article belongs to.
+ * @param {string} [organizationId=null] The selected organization's ID to create the article under.
+ * @param {'v1'|'v2'} [editorVersion='v2'] The editor version.
+ *
+ * @returns {Cypress.Chainable<Cypress.Response>} A cypress request for creating an.
+ */
+Cypress.Commands.add(
+  'createArticle',
+  ({
+    title,
+    tags = [],
+    content = '',
+    published = true,
+    description = '',
+    canonicalUrl = '',
+    series = '',
+    allSeries = [],
+    organizations = [],
+    organizationId = null,
+    editorVersion = 'v2',
+  }) => {
+    return cy.request('POST', '/articles', {
+      article: {
+        id: null,
+        title,
+        tagList: tags.join(','),
+        description,
+        canonicalUrl,
+        series,
+        allSeries,
+        bodyMarkdown: content,
+        published,
+        submitting: false,
+        editing: false,
+        mainImage: null,
+        organizations,
+        organizationId,
+        edited: true,
+        updatedAt: null,
+        version: editorVersion,
+        siteLogo:
+          '<a href="/" class="site-logo" aria-label="DEV(local) Community Home">\n    <span class="truncate-at-2">\n      DEV(local) Community\n    </span>\n</a>\n',
```

Comment:
When I inspected the Network tab to create this custom command, the request sent the site's logo is part of the new article creation. Any reason this is required? cc: @rhymes

Reply:
I'm not very familiar with the Article's editor to be honest. That's weird. It looks like it's HTML embedded in a `data-` attribute 😕 

https://github.com/forem/forem/blob/11da7ddb9f09d55c35bddb9ecf866e18942adaa5/app/views/articles/_v2_form.html.erb#L43

It doesn't seem to be fetched in JS though. I wonder if it's a remnant of something older. It comes from https://github.com/forem/forem/pull/10840/files#diff-6f417f63e88e9b7e4b28f32d77d1d6ed33fd8d0500c7494e29fdc31bfc7b482bR20

Reply:
I removed `siteLogo`, `helpFor` and `helpPosition` and article creation still works. Regardless, still odd that it's all being sent.